### PR TITLE
feat(utils): add file size logging

### LINKED ISSUE #59: Log report file size
It helps to understand the size of produced artefacts. Let' log it.

### PR DESCRIPTION
**Pull request by BioPhoton**
This PR includes:
- formatBytes function and tests
- logPersisted files function and tests
- log the files in the CLI `collect` command

closes #59

### CODE REVIEW

**Review by vmasek — APPROVED** [review]

**Review by matejchalk — APPROVED** [review]

**Review by matejchalk — APPROVED** [review]

**Review by vmasek — APPROVED** [review]

File: packages/utils/src/lib/collect/implementation/utils.spec.ts [hunk 1]
```diff
@@ -60,3 +60,49 @@ describe('sumRefs', () => {
...
+    expect(formatBytes(10000000000000000000000000)).toBe('8.27 YB');
+  });
+
+  it('should log file sizes correctly with correct decimal`', async () => {
+    expect(formatBytes(10000, 1)).toBe('9.8 kB');
+  });
+
+  it('should log file sizes of 0 if no size is given`', async () => {
+    expect(formatBytes(0)).toBe('0 B');
+  });
```
Comment (ChristopherPHolder) [inline packages/utils/src/lib/collect/implementation/utils.spec.ts:107]: Is the expected behavior that it logs a file with 0 bites ?

File: package.json [hunk 2]
```diff
@@ -52,6 +52,9 @@
     "vitest": "~0.32.0"
   },
   "optionalDependencies": {
+    "@esbuild/darwin-arm64": "^0.19.4",
+    "@nx/nx-darwin-arm64": "^16.9.1",
+    "@nx/nx-darwin-x64": "^16.9.1",
```
Comment (ChristopherPHolder) [inline package.json:57]: If this is 64 because of the architecture we also need to include the 32 arch

**Review by ChristopherPHolder — APPROVED** [review]
Both things are minor and I agree that if all test are passing the PR can be merged

File: packages/utils/src/lib/collect/implementation/utils.spec.ts (hunk 1 above)
Comment (BioPhoton) [reply packages/utils/src/lib/collect/implementation/utils.spec.ts:107]: It gets a number passed for display logic. So I was refrained to throw here.

**Pull request merged**